spec/koans.js

```diff
@@ -423,21 +435,27 @@ describe("the JavaScript language", function() {
       var obj = createObject();
       obj.addPoint();
 
-      //expect(obj.score()).toEqual();
-      //expect(typeof(obj.points)).toEqual();
+      expect(obj.score()).toEqual(1);
+      expect(typeof(obj.points)).toEqual('undefined');
     });
 
     it("may create objects also with the module pattern", function() {
-      function createObject(initialScore) {
-        // write the code here
+      function createObject(initialScore, arg) {
+        this.initialScore = initialScore;
+        this.color = arg;
+        this.incrementScoreIn = function(initialScore){return this.initialScore + initialScore}
+
+        return {
+          points: function(){return this.incrementScoreIn(5); },
```

Comment:
esta linea dentro del objeto solucionaria el test en el que se quedaron "incrementScoreIn: function(s){this.score += s}," estan accediendo a un método desde un objeto. chequen eso